foremwebview/build.gradle

```diff
@@ -65,7 +68,7 @@ afterEvaluate {
 
         groupId = 'com.github.forem'
         artifactId = 'ForemWebView-android'
-        version = '1.0.10'
+        version = '$version_name'
```

Comment:
@Chandra-Sekhar-Bala This one seems incorrect because this is not using variable its just a string.

This PR is a bit tricky because the changes cannot be fully confirmed until and unless I create a release for this library. But in any case I think the most correct guess here would be "$version_name"

Any thoughts?

Reply:
Thanks for pointing that out. Yes,  using "$version_name" would be great for now, as they share the same interest. If anything breaks after release we can fix that quickly.

Reply:
Can you please commit this change, so that I can approve it and merge it. @Chandra-Sekhar-Bala

Reply:
Sorry think I don't get it properly, are you asking to change:
```version = '$version_name'```  to this ```version = '1.1.10'``` or just mark this conversation as Resolved?

Reply:
Okay, thanks :)